guides/v2.2/config-guide/cli/config-cli-subcommands-cache.md

```diff
@@ -168,15 +180,20 @@ Flush cache types if you've already tried cleaning the cache and you're still ha
 
 Command usage:
 
-	bin/magento cache:clean [type] ... [type]
-	bin/magento cache:flush [type] ... [type]
+```bash
+bin/magento cache:clean [type] ... [type]
+bin/magento cache:flush [type] ... [type]
+```
 
 Where `[type]` is a space-separated list of cache types. Omitting `[type]` cleans or flushes all cache types at the same time. For example, to flush all cache types, enter
 
-	bin/magento cache:flush
+```bash
+bin/magento cache:flush
+```
 
 Sample result:
 
+```terminal
```

Comment:
Is `terminal` a correct alias for syntax highlight? I'm unable to find it in the [linguist](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) reference which is used by Github. 
Maybe we should use `bash` in such cases as well?

Reply:
We use `bash` for commands because the CSS adds the $ to the line but it is not included when copying.
`terminal` will work and it is used to show the output/results of a command. It will not include the $ at the start of the line.